pom.xml

```diff
@@ -197,9 +197,9 @@
     </extensions>
     <plugins>
       <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>findbugs-maven-plugin</artifactId>
-        <version>${findbugs-maven-plugin.version}</version>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <version>${spotbugs-maven-plugin.version}</version>
         <configuration>
           <excludeFilterFile>${session.executionRootDirectory}/buildtools/src/main/resources/bookkeeper/findbugsExclude.xml</excludeFilterFile>
```

Comment:
Should probably rename this file also

Reply:
@ivankelly there is already an issue, I have already sent emails in the past months about this change.
I think we change keep findbugsExclude.xml as effectively SpotBugs is Findbugs (it just changed owner) and internally the format of the file is the same. The SportBugs maven plugin is a port of the Findbugs Maven plugin and all the configs are the same.

But if you feel strong about this we can change it as well.

Reply:
@eolivelli it still needs to be announced when done, so people know they need to run spotbugs:check instead of findbugs:check

Reply:
filename change isn't important, i'd just update it if other things need an update